services/web-server/src/monitor.js

```diff
@@ -4,4 +4,17 @@ const manager = new MonitorManager({
   serviceName: 'web-server',
 });
 
+manager.register({
+  name: 'createCredentials',
+  title: 'Credentials Created',
+  type: 'create-credentials',
+  version: 1,
+  level: 'info',
+  description: 'A user has been issued Taskcluster credentials',
+  fields: {
+    credentials: 'Taskcluster credentials.',
```

Comment:
I suspect we don't want to log the full set of credentials -- probably just the clientId.